Makefile

```diff
@@ -8,18 +8,27 @@ all: image
 container: image
 
 image:
-	docker build --build-arg VERSION=$(VERSION) -t $(IMAGE_NAME) . # Build image and automatically tag it as latest
+	docker build --build-arg VERSION=$(VERSION) --build-arg BASE_TOMCAT_IMAGE=tomcat:9-jdk17 -t $(IMAGE_NAME) . # Build image and automatically tag it as latest on jdk17
+	docker build --build-arg VERSION=$(VERSION) --build-arg BASE_TOMCAT_IMAGE=tomcat:9-jdk11 -t $(IMAGE_NAME):3-jdk11 . # Build image using tomcat 9 , jdk 11
+	docker build --build-arg VERSION=$(VERSION) --build-arg BASE_TOMCAT_IMAGE=tomcat:9-jdk17 -t $(IMAGE_NAME):3-jdk17 . # Build image using tomcat 9 , jdk 17
```

Comment:
We can probably just build the jdk17 image once and retag it using `docker tag`

Reply:
This can be done for sure. But in the future latest can be build on any higher jdk version also.
Also I am running java version test cases on the `latest` tag to ensure it is correctly build on jdk17.

Retagging is happening in `push` task that is the very last action of the build and before that we want to certify java versions of all the different tags. 

Building latest tag separately gives more visibility about the jdk version it is using.